### PR TITLE
Exit with non-zero status for non-rake interface

### LINKED ISSUE #47: bin/coffeelint.rb app.coffee doesn't exit with non-zero status when offenses are found
Hello! And thanks for this lovely gem.

I recently switched to the binstub interface for running coffeelint and discovered that some lints were sneaking thru our continuous integration process. It seems that this interface doesn't use an exit code to indicate whether there were any lints present.

### PR DESCRIPTION
**Pull request by maxjacobson**
fix #47

### CODE REVIEW

File: lib/coffeelint/cmd.rb [hunk 1]
```diff
@@ -79,13 +79,16 @@ def self.main
       options = parse_options
 
       if ARGV.length > 0
+        errors_count = 0
         ARGV.each do |file|
           if options[:options][:recursive]
             Coffeelint.run_test_suite(file, options[:linter_options])
```
Comment (zmbush) [inline lib/coffeelint/cmd.rb]: We should probably do the same thing for `run_test_suite`

**Comment by zmbush** [comment]
This looks good, just add the change for run_test_suite and I'll merge.

**Comment by maxjacobson** [comment]
Ah, great point. Updated!

**Comment by zmbush** [comment]
Looks good. Thanks!

**Comment by maxjacobson** [comment]
:smile: my pleasure, and thanks

**Pull request merged**